tests/unit/src/test/scala/tests/JarTopLevelsSuite.scala

```diff
@@ -0,0 +1,50 @@
+package tests
+
+import scala.collection.concurrent.TrieMap
+import java.nio.file.Paths
+
+import scala.meta.io.AbsolutePath
+import scala.meta.internal.metals.JarTopLevels
+import scala.meta.internal.io.PlatformFileIO
+
+object JarTopLevelsSuite extends BaseTablesSuite {
+  def jarSymbols: JarTopLevels = tables.jarSymbols
+  def getAbsolutePath(resource: String): AbsolutePath = {
+    AbsolutePath(Paths.get(getClass.getResource(resource).toURI))
+  }
+  val jar1 = getAbsolutePath("/jar-symbols/jar1.zip")
```

Comment:
What are the contents of these zip files? Can we generate them programmatically instead of committing them into the repo?

Alternatively, in #527 we add a utility to download sources from GitHub https://github.com/scalameta/metals/blob/5077bdaefbf7cb1219b538fb130e60d27cb5d163/tests/unit/src/main/scala/bench/Corpus.scala

Reply:
Example code to create zip file using our IO utils

```scala
import java.nio.charset.StandardCharsets
import java.nio.file.Files
import scala.meta.internal.io.FileIO
import scala.meta.io.AbsolutePath
    val tmp =
      AbsolutePath(Files.createTempDirectory("metals")).resolve("sources.zip")
    FileIO.withJarFileSystem(tmp, create = true, close = true) { root =>
      FileLayout.fromString(
        """|/foo.scala
           |object Hello {
           |}""".stripMargin,
        root
      )
    }
```

Reply:
Done, generating test zip files, removed the static zip files from test resources.